adal-connectivity/src/main/java/com/massivedisaster/adal/connectivity/NetworkUtils.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * ADAL - A set of Android libraries to help speed up Android development.
+ *
+ * Copyright (c) 2017 ADAL
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+ * LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package com.massivedisaster.adal.connectivity;
+
+import android.content.Context;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+
+/** Utility class for the Network sub module. */
+@SuppressWarnings({"unused", "WeakerAccess"}) public final class NetworkUtils {
+
+    public static final int TYPE_NOT_CONNECTED = 0;
+    public static final int TYPE_WIFI = 1;
+    public static final int TYPE_MOBILE = 2;
+
+    /** Instantiates a new NetworkUtils. Private to prevent instantiation. */
+    private NetworkUtils() {
+        // Throw an exception if this *is* ever called
```

Comment:
@paribeiro10 as opposed to https://github.com/massivedisaster/ADAL/pull/125/files#r137399198 the comment is present and above the throw.

Reply:
I removed it as it is not needed. Well spotted on that one.